### DISCUSSION TITLE
Increase Opacity of "Sort By" dropdown in song select

### DISCUSSION THREAD

**Discussion by ampersand64** [ppy/osu#29906] [Ideas] [answered]
The current background opacity of the "Sort By" dropdown in song select has low readability, mostly against light-colored beatmap backgrounds. 

![osu_2024-09-17_15-13-37](https://github.com/user-attachments/assets/e0889fe3-ba7c-43cc-9ff4-65ec0f05fd75)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/29912
